packages/react-dom/src/__tests__/ReactServerRendering-test.js

```diff
@@ -632,7 +667,7 @@ describe('ReactDOMServer', () => {
       });
     });
 
-    it('should refer users to new apis when using suspense', async () => {
+    it('should omit text and suspense placeholders', async () => {
```

Comment:
This test is now asserting the removal of hydration markup even though I kept the test form largely intact